ext/openssl/tests/openssl-get-cipher-methods.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+array openssl_get_cipher_methods ([ bool $aliases = false ] );
+--CREDITS--
+marcosptf - <marcosptf@yahoo.com.br>
+--SKIPIF--
+<?php 
+if (!extension_loaded("openssl")) print "skip";
+if (OPENSSL_VERSION_NUMBER < 0x10000000) die("skip Output requires OpenSSL 1.0");
```

Comment:
why? you can of course use it in any OpenSSL version that we support so pls remove this skip